### PR TITLE
add discord column to admin and committees

### LINKED ISSUE #741: Add Discord Column
image

- Make sure to add it to the form
- Add discord column in table

### LINKED ISSUE #742: Add Discord Column
image

- Refer to the Admins Discord Column

### PR DESCRIPTION
**Pull request by NovaSagittarii**
hi discord column is added (i cant check committees since those don't exist apparently)

image

image

tfw all the api routes start looking the same :thinking:

### CODE REVIEW

File: src/app/api/admins/route.js [hunk 1]
```diff
@@ -58,12 +58,13 @@ export async function GET() {
...
     );
     snapshot.forEach((doc) => {
-      const { name, email, roles, affiliation } = doc.data();
+      const { name, email, roles, affiliation, discord } = doc.data();
 
       output.push({
         uid: doc.id,
         name: name,
         email: email,
+        discord,
```
Comment (shahdivyank) [inline src/app/api/admins/route.js]: can u match the code style in the file

File: src/app/api/committees/route.js [hunk 2]
```diff
@@ -62,11 +62,12 @@ export async function GET() {
       )
     );
     snapshot.forEach((doc) => {
-      const { name, email, roles, affiliation } = doc.data();
+      const { name, email, roles, affiliation, discord } = doc.data();
       output.push({
         uid: doc.id,
         name: name,
         email: email,
+        discord,
```
Comment (shahdivyank) [inline src/app/api/committees/route.js]: match code style

File: src/components/admin/dashboards/Toolbar.jsx [hunk 3]
```diff
@@ -48,7 +48,6 @@ const Toolbar = ({
 
   const handleSubmit = (e) => {
     e.preventDefault();
-
```
Comment (shahdivyank) [inline src/components/admin/dashboards/Toolbar.jsx]: revert this

File: src/app/api/admins/route.js (hunk 1 above)
Comment (NovaSagittarii) [reply src/app/api/admins/route.js]: ok

File: src/app/api/committees/route.js (hunk 2 above)
Comment (NovaSagittarii) [reply src/app/api/committees/route.js]: ok

File: src/components/admin/dashboards/Toolbar.jsx (hunk 3 above)
Comment (NovaSagittarii) [reply src/components/admin/dashboards/Toolbar.jsx]: ok

File: src/components/admin/dashboards/Toolbar.jsx [hunk 4]
```diff
@@ -48,11 +48,9 @@ const Toolbar = ({
...
     setObjects(
       objects.map((a) => {
         let boolean = false;
-
```
Comment (shahdivyank) [inline src/components/admin/dashboards/Toolbar.jsx]: revert this

File: src/components/admin/dashboards/Toolbar.jsx (hunk 3 above)
Comment (shahdivyank) [reply src/components/admin/dashboards/Toolbar.jsx]: revert

Comment (NovaSagittarii) [reply src/components/admin/dashboards/Toolbar.jsx]: wait wat i didnt?

File: src/components/admin/dashboards/Toolbar.jsx (hunk 4 above)
Comment (NovaSagittarii) [reply src/components/admin/dashboards/Toolbar.jsx]: aaaaaaaaaaaaaaaaaaaa

**Review by shahdivyank — APPROVED** [review]

**Review by SafeDuck — APPROVED** [review]

**Pull request merged**